specs/trs/tracked-resource-set.html

```diff
@@ -423,15 +423,28 @@ <h2>Base Segmentation</h2>
       <p>
         The number of Tracked Resources in the Base could be such that it is not reasonable to list them all in a single
         HTTP response. In this case,
-        <span class="conformance"
-          >the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
-          listing a subset of all Tracked Resources</span
-        >.
+        <span class="conformance">
+          the Base MAY be segmented into separate resources; there can be any number of such Base segments, each one
+          listing a subset of all Tracked Resources.</span
+        >
+      </p>
+      <p class="conformance">
+        If base paging is implemented, the server MAY respond with a 30x redirect message, directing the Client to the
+        first “single-page resource”, or alternatively it MAY respond with the first "single-page resource" .
       </p>
       <p class="conformance">
-        Base segmentation MUST be accomplished using the mechanisms described in the "Resource Paging" section of
-        [[!OSLCCore3]].
+        The representation of a single-page resource MAY contain a subset of the Base’s membership triples.
+      </p>
+      <p>
+        <span class="conformance"
+          >The response for a single-page resource SHOULD contain a Link: &lt;http://www.w3.org/ns/ldp#Page&gt;;
```

Comment:
It's actually a MUST in LDP paging https://www.w3.org/TR/ldp-paging/#h-ldpr-pagingget-page-type-reqd

I expect non-controversial clauses from LDP paging to be mirrored as long as OSLC aims to reuse LDP properties. Otherwise, we'd be splitting semantics.

Reply:
We are not implementing or referencing LDP paging. Forget that LDP paging exists. It's not relevant heree.

Reply:
Well, then don't use `http://www.w3.org/ns/ldp#Page` if this proposal has nothing to do with LDP and define `oslc_trs:Page` instead.

Reply:
We cannot do that. It would be a breaking change. It must be that link for existing consumers of paged base to work. A reminder. The proposal is about maintaining compatibility with the TRS 2.0 spec and existing server and client implementations.

Reply:
I know that, of course. That's why I first proposed to bring the semantics of the subset of LDP that is used here in line with the LDP paging WD. I think we are talking about 2 MUSTs in total. But after 10 years of fun with SHOULDs and MAYs, here we go again, in the process watering down the semantics of predicates under stewardship of W3C, not OASIS.

Reply:
David, Andrew was responding to my request to review the changes since I knew he had some reservations and due to recent lack of availability hasn't been involved in the followup on this issue. The responses above are exactly what I was looking for - confirmation that Andrew's concerns were noted and addressed before we submit for PS02. Thanks to both of you for the followup.